<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_main</class>
 <widget class="QMainWindow" name="GUI_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI_main</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="View_table">
     <attribute name="title">
      <string>实时监测</string>
     </attribute>
     <widget class="QLabel" name="Cam1_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>401</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Cam3_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>220</y>
        <width>401</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Cam2_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>401</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Cam0_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="State_1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>470</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>流程 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="State_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>470</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>流程 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="State_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>470</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>流程 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="State_4">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>470</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>流程 4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Chage_1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>520</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>换班</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Chage_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>520</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>换班</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Chage_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>520</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>换班</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Chage_4">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>520</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>换班</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Start_info">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>470</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>批次信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="Info_Cam1_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> Cam1</string>
      </property>
     </widget>
     <widget class="QLabel" name="Info_Cam3_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>220</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> Cam3</string>
      </property>
     </widget>
     <widget class="QLabel" name="Info_Cam2_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> Cam2</string>
      </property>
     </widget>
     <widget class="QLabel" name="Info_Cam0_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> Cam0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Search_table">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <attribute name="title">
      <string>订单查找</string>
     </attribute>
     <widget class="QLabel" name="Info_Cam1">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> Cam1</string>
      </property>
     </widget>
     <widget class="QLabel" name="Info_Cam0">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> Cam0</string>
      </property>
     </widget>
     <widget class="QLabel" name="Cam2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>401</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Info">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>71</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Info_Cam3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>220</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> Cam3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>440</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>序号输入</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Search">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>510</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="Input">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>470</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Cam0">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Cam3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>220</y>
        <width>401</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Info_Cam2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> Cam2</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="Datebase">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>450</y>
        <width>631</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Cam1">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>401</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="View">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>540</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>监测</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
